src/main/kotlin/apply/application/ApplicantService.kt

```diff
@@ -0,0 +1,50 @@
+package apply.application
+
+import apply.domain.applicant.Applicant
+import apply.domain.applicant.ApplicantRepository
+import apply.domain.applicant.Gender
+import org.springframework.stereotype.Service
+import org.springframework.transaction.annotation.Transactional
+import javax.annotation.PostConstruct
+
+@Transactional
+@Service
+class ApplicantService(val applicantRepository: ApplicantRepository) {
```

Comment:
`ApplicantService` 외에 사용하는 곳이 없어 보이네요. public이 아닌 private으로 바꾸면 어떨까요?

```suggestion
class ApplicantService(private val applicantRepository: ApplicantRepository) {
```

Reply:
수정했습니다